app/views/projects.html

```diff
@@ -97,27 +97,29 @@ <h2 class="text-center">Loading...</h2>
               <div class="list-group list-view-pf projects-list">
                 <div ng-repeat="project in projects | limitTo: limitListTo track by (project | uid)" class="list-group-item project-info tile-click">
                   <div class="list-view-pf-main-info">
-                    <div class="list-view-pf-description project-names">
-                      <div class="list-group-item-heading project-name-item">
-                        <h2 class="h1">
-                          <a class="tile-target" ng-href="project/{{project.metadata.name}}" title="{{project | displayName}}"><span ng-bind-html="project | displayName | highlightKeywords : keywords"></span></a>
-                          <span ng-if="project.status.phase != 'Active'"
-                            data-toggle="tooltip"
-                            title="This project has been marked for deletion."
-                            class="pficon pficon-warning-triangle-o"></span>
-                        </h2>
-                        <small>
+                    <div class="list-view-pf-body">
+                      <div class="list-view-pf-description">
+                        <div class="list-group-item-heading project-name-item">
+                          <span class="h3">
```

Comment:
span is semantically neutral, so this change actually removes semantic meaning.  It's better as an h2.